pallets/ajuna-gameregistry/src/lib.rs

```diff
@@ -225,6 +238,11 @@ pub mod pallet {
 
 				game.winner = Some(winner);
 
+				// Players need to know which game they are in
```

Comment:
removing reference of game from player.

Reply:
which line are you trying to reference?  Not sure what you are trying to say here, sorry.